Transformer/Tests/ConditionFromBooleanToMongoTransformerTest.php

```diff
@@ -1,93 +0,0 @@
-<?php
```

Comment:
@itkg-nanne : ok, let's do as usual then, but I suggest to discuss this point later.